core/src/main/resources/hudson/model/Job/_api.jelly

```diff
@@ -45,23 +45,24 @@ THE SOFTWARE.
 
   <h2>Perform a build</h2>
   <p>
-    To programmatically schedule a new build, post to <a href="../build">this URL</a>. 
+    To programmatically schedule a new build, post to <a href="../build?delay=0sec">this URL</a>.
```

Comment:
```suggestion
    To programmatically schedule a new build, post to <a href="../build">this URL</a>.
    Optionally add a <code>delay</code> query parameter to override the default quiet period, e.g., <code>delay=0</code> (in milliseconds, no delay) or <code>delay=60sec</code> (in seconds, one minute).
```

Reply:
I think in practice you will almost always want `?delay=0sec` in the REST context. That is why the button passes this. If you are asking to trigger a build, why wait? The quiet period is really only there for SCM triggers.